src/can-stache-element.js

```diff
@@ -64,6 +66,45 @@ function DeriveElement(BaseElement = HTMLElement) {
 	StacheElementConstructorFunction.prototype = Object.create(StacheElement.prototype);
 	StacheElementConstructorFunction.prototype.constructor = StacheElementConstructorFunction;
 
+	// We can't set `observedAttributes` on the `StacheElement.prototype` as static properties are
+	// not copied over with `Object.create`
+	Object.defineProperty(StacheElementConstructorFunction, 'observedAttributes', {
```

Comment:
Would it make sense to add a function to `mixin-binding-prop.js` that does this? Just so the code could be co-located with the `initialize` method that uses the `_uninitializedBindings`.

Reply:
Sure that makes sense.